IdentityServer/v7/docs/content/diagnostics/otel/metrics.md

```diff
@@ -7,66 +7,74 @@ weight: 10
 (added in v7.0)
 
 The metric counters are designed to provide a high level overview. They are designed to not contain any
-sensitive information. The counters may contain tags to indicate the source of the data.
+sensitive information. The counters may contain tags to indicate the source of the data. Metrics are
+intended to be used to provide overall health metrics and are typically used to show graphs on
```

Comment:
I think we should move the remarks about sensitive data and tags later (see below), and start off with a statement of what a metric is.
```suggestion
OpenTelemetry metrics are counters designed to provide a high level overview of the frequency events that are occurring in your system. 
Metrics are intended to be used to provide overall health metrics and are typically used to show graphs on
```